update_test.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 func TestDynamicUpdateParsing(t *testing.T) {
+	// This does nothing with dynamic updates, move elsewhere?
```

Comment:
I agree this function should be tested, but in zscan_test.go as it is defined in zscan.go. All it does here is parse partial RRs, which is fine but not specific to updates.go. In fact I can parse partial RRs everywhere I want ^^

But never mind, it is not worth the discussion, al long as it is tested.